tools/docker-images/clp-execution-base-ubuntu-noble/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM ubuntu:noble AS base
+
+WORKDIR /root
+
+RUN mkdir -p ./tools/docker-images/clp-execution-base-ubuntu-noble
+ADD ./tools/docker-images/clp-execution-base-ubuntu-noble/setup-scripts ./tools/docker-images/clp-execution-base-ubuntu-noble/setup-scripts
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)